CHANGELOG.md

```diff
@@ -27,17 +30,24 @@ All notable changes to this project will be documented in this file.
   - Improved naming for AMQP binding queues
   - Shelve VMs
   - Neutron Networking
+  - No longer using adminURL
 - Foreman: Exposed additional properties to reporting
 - Azure
   - Initial work for Inventory Collection, OAuth2, [azure-armrest gem](https://github.com/ManageIQ/azure-armrest)
   - Azure Provider models
   - Power Operations
+  - Added as a type for Cloud Provider discovery
+  - User Image Inventory
+  - Provision from private user image with template
 - RHEVM: Reconfigure of Memory and CPU
 - Orchestration: Reconfiguration
 - Reporting on Providers
   - Ability to report on Performance
   - Host Socket and Total VMs metrics
   - Watermark reports available out-of-the-box
+- Google
+  - Community addition
+  - Work has begun on this provider  
```

Comment:
I was wondering if it should be here at all at this point.

--M

On Wed, Nov 25, 2015 at 1:38 PM, Jason Frey <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/ManageIQ/manageiq/pull/5585#discussion_r45903472>:
>
> >  - RHEVM: Reconfigure of Memory and CPU
> >  - Orchestration: Reconfiguration
> >  - Reporting on Providers
> >    - Ability to report on Performance
> >    - Host Socket and Total VMs metrics
> >    - Watermark reports available out-of-the-box
> > +- Google
> > +  - Community addition
> > +  - Work has begun on this provider
>
> This is weird from a CHANGELOG perspective...sounds more like a slide ;)
> @blomquisg <https://github.com/blomquisg> Thoughts here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/5585/files#r45903472>.
>



-- 
Marianne Feifer
Senior Technical Project Manager
CloudForms Management Engine
mfeifer@redhat.com
irc: mfeifer
freenode: marigold